wiki/Main_Page/id.md

```diff
@@ -120,7 +120,7 @@ Organisasi: [osu! UCI](/wiki/Community/Organisations/osu!_UCI)
 
 # Hal-hal terkait pengembangan
```

Comment:
This can be worded as
```suggestion
# osu! untuk developer
```

Example: [Facebook untuk Developer](https://developers.facebook.com/?locale=id_ID)

Reply:
hmm i cross-checked this w/ the original English article and apparently the original string here is simply `"For developers"`, so i simplified it into `Untuk para pengembang` instead